src/otx/recipe/detection/rtdetr_101.yaml

```
@@ -0,0 +1,115 @@
+model:
+  class_path: otx.algo.detection.rtdetr.OTXRtDetr101
+  init_args:
+    label_info: 80
+
+    optimizer:
+      class_path: torch.optim.AdamW
+      init_args:
+        lr: 0.0001
+        betas: [0.9, 0.999]
+        weight_decay: 0.0001
+
+    scheduler:
+      class_path: otx.core.schedulers.LinearWarmupSchedulerCallable
+      init_args:
+        num_warmup_steps: 5
+        main_scheduler_callable:
+          class_path: lightning.pytorch.cli.ReduceLROnPlateau
+          init_args:
+            mode: max
+            factor: 0.1
+            patience: 6
+            monitor: val/map_50
+
+engine:
+  task: DETECTION
+  device: auto
+
+callback_monitor: val/map_50
+
+data: ../_base_/data/torchvision_base.yaml
+overrides:
+  callbacks:
+    - class_path: otx.algo.callbacks.adaptive_train_scheduling.AdaptiveTrainScheduling
+      init_args:
+        max_interval: 1
+        decay: -0.025
+        min_lrschedule_patience: 3
+    - class_path: otx.algo.callbacks.adaptive_early_stopping.EarlyStoppingWithWarmup
+      init_args:
+        monitor: null
+        mode: max
+        patience: 10
+        check_on_train_epoch_end: false
+        min_delta: 0.001
+        warmup_iters: 100
+        warmup_epochs: 7
+  data:
+    task: DETECTION
+    stack_images: True
+    data_format: coco_instances
+    train_subset:
+      batch_size: 4
+      to_tv_image: True
+      transforms:
+        - class_path: torchvision.transforms.v2.RandomPhotometricDistort
+          init_args:
+            p: 0.5
+        - class_path: torchvision.transforms.v2.RandomZoomOut
+          init_args:
+            fill: 0
+        - class_path: otx.core.data.transform_libs.torchvision.RandomFlip
+          init_args:
+            prob: 0.5
+        - class_path: otx.core.data.transform_libs.torchvision.Resize
+          init_args:
+            scale:
+              - 640
+              - 640
+            keep_ratio: false
+            transform_bbox: true
+            is_numpy_to_tvtensor: true
+        - class_path: torchvision.transforms.v2.ToDtype
+          init_args:
+            dtype: ${as_torch_dtype:torch.float32}
+            scale: True
+        - class_path: torchvision.transforms.v2.SanitizeBoundingBoxes
+        - class_path: otx.core.data.transform_libs.torchvision.ConvertBox
+          init_args:
+            out_fmt: "cxcywh"
+            normalize: True
+      sampler:
+        class_path: otx.algo.samplers.balanced_sampler.BalancedSampler
+    val_subset:
+      batch_size: 8
```

```suggestion
      sampler:
        class_path: otx.algo.samplers.balanced_sampler.BalancedSampler
        
    val_subset:
      batch_size: 8
```
Could you add empty line between second level groups or (train, val, test) subsets?

---

Done https://github.com/openvinotoolkit/training_extensions/pull/3741/commits/bda9a9d05e96151cabcbdb13320460715740b87f